packages/protocol/contracts/governance/Validators.sol

```diff
@@ -451,6 +451,7 @@ contract Validators is
     // The group that should be paid is the group that the validator was a member of at the
     // time it was elected.
     address group = getMembershipInLastEpoch(account);
+    require(group != address(0), "Validator not registered with a group");
```

Comment:
If this line fails, reward distribution will stop. I've created https://github.com/celo-org/celo-blockchain/pull/881 to help address that and it should be merged along with this.